app/routes/questionnaire.py

```diff
@@ -85,19 +85,21 @@ def before_questionnaire_request():
     if questionnaire_store.submitted_at:
         return redirect(url_for("post_submission.get_thank_you"))
 
-    logger.bind(
+    contextvars.bind_contextvars(
         tx_id=metadata.tx_id,
         ce_id=metadata.collection_exercise_sid,
     )
 
     if schema_name := metadata.schema_name:
-        logger.bind(schema_name=schema_name)
+        contextvars.bind_contextvars(schema_name=schema_name)
 
     if schema_url := metadata.schema_url:
-        logger.bind(schema_url=schema_url)
+        contextvars.bind_contextvars(schema_url=schema_url)
 
-    logger.info(
-        "questionnaire request", method=request.method, url_path=request.full_path
+    contextvars.bind_contextvars(
```

Comment:
How come this has changed from `logger.info`?

Reply:
I got carried away here and started adding some unnecessary logger changes, I compared it with master and the logs are working as expected without this change so reverted it.